### ISSUE TITLE
On linux chrome, github highlighted codeblock horizontal scrollbar can overlap with the next msg and get lost

### ISSUE THREAD

**Issue by matrixbot** [vector-im/vector-web#754]
Created by @ matthew:matrix.org.

**Comment by ara4n** [comment]
See https://matrix.org/_matrix/media/v1/download/jki.re/niqajBJeKEKZKMJwcRGpoWbh

**Comment by erikjohnston** [comment]
Removing the following removes the bug (but changes layout):

```css
.mx_EventTile_continuation {
    margin-top: 8px!important;
}
```

**Comment by ara4n** [comment]
also "Chrome 50.0.2638.0 on Windows 7 SP1 -- the horizontal scroll bar on those little "code" blocks with the error log snippets doesn't work for me. It does not drag to the right. But if I hold the mouse button down and then select the contents from the top left to the bottom right, it does both select and scroll to the right."

**Comment by ara4n** [comment]
fixed by 490cbbd
